scripts/integration.sh

```diff
@@ -10,13 +10,13 @@ docker-compose -f docker-compose.yml -f docker-compose.prod.yml up -d
 trap "docker-compose down" INT EXIT
 
 set +e
-wget -t 60 --spider --retry-connrefused --waitretry=1  0.0.0.0:443 || sleep 2
-wget -t 60 --spider --retry-on-http-error=404  1 0.0.0.0:443/v1/directories/default
+wget -T 60 --spider --retry-connrefused localhost:443
+wget -T 60 --spider --retry-on-http-error=404 --no-check-certificate https://localhost/v1/directories/default
```

Comment:
Are you sure you *don't* want to fail the script here if you can't get a 200 response? It seems it'd make it easier to tell what went wrong. Is it just because you've found it to be a bit unreliable?

Reply:
Fixed